### DISCUSSION TITLE
Word origin of "Hasura"

### DISCUSSION THREAD

**Discussion by eggplants** [hasura/graphql-engine#8518] [Question] [answered]
I would like to know the word origin of hasura. I guess hasura is named after hustler. Does any source or documentation exist?

**Comment by arunkumar-patange** [comment, marked as answer]
https://mobile.twitter.com/hasurahq/status/1395355538041360386

**Reply by eggplants** [reply]
>👹 ASURA - The Sanskrit word for demon & a reference to 'daemons' in computer programming
>👹 H - From Haskell, the language our GraphQL to SQL compiler is written in

That makes sense why harusa logo is λ + 👹

**Comment by seanonthenet** [comment]
Wish it was Rustura 😉😉😉

Juuust kidding.

**Reply by maaft** [reply]
Or Gosura...
